genesyscloud/resource_genesyscloud_processautomation_trigger.go

```diff
@@ -285,17 +303,14 @@ func updateProcessAutomationTrigger(ctx context.Context, d *schema.ResourceData,
 			return resp, diag.Errorf("Failed to read process automation trigger %s: %s", d.Id(), getErr)
 		}
 
-		//make sure that topic name is not updated
-		if topic_name != *trigger.TopicName {
```

Comment:
I totally missed this in the initial review. Yeah we should never do this. Always let `ForceNew` take care of it